go/src/koding/klientctl/main.go

```diff
@@ -743,6 +744,26 @@ error opening: %s
 					},
 				},
 			}, {
+				Name:        "sync",
+				Usage:       "Manage mount synchronization.",
+				Description: cmdDescriptions["mount-sync"],
+				Action:      ctlcli.ExitErrAction(MachineSyncMount, log, "sync"),
+				Flags: []cli.Flag{
+					cli.DurationFlag{
+						Name:  "timeout, t",
+						Usage: "Maximum time to wait.",
+						Value: time.Minute,
+					},
+					cli.BoolFlag{
+						Name:  "pause",
```

Comment:
I still think they should be subcommands and not flags. Can we pick up this issue elsewhere?